src/migration.mjs

```diff
@@ -0,0 +1,66 @@
+import { PrismaClient } from "@prisma/client";
+import * as fs from 'fs';
+
+const prisma = new PrismaClient();
+
+async function readSpreadsheet(filePath) {
+    try {
+      const buffer = fs.readFileSync(filePath);
+      const data = await toCSV(buffer);
+      await processExcelData(data)
+    } catch (error) {
+      console.log(error);
+    } finally {
+      await prisma.$disconnect();
+    }
+}
+
+async function toCSV(buffer) {
+  const lines = [];
+  for (let i = 0; i < buffer.length; i++) {
```

Comment:
this whole bit should be expressable as `const csvString = buffer.map(i => i.toString()).join(",)` instead of a for - that way you dont have to worry about tracking the index variable